take-homes/be-take-home.md

```diff
@@ -0,0 +1,48 @@
+# Tea Subscription Service
+
+You will create a Rails API for a Tea Subscription Service.
+
+## Requirements:
+
+At a minimum, you must create:
+
+- An endpoint to subscribe a customer to a tea subscription
+- An endpoint to cancel a customer's tea subscription
+- An endpoint to see all of a customer's subsciptions (active and cancelled)
+
+Beyond the base requirements, include any additional endpoints you'd like, but be mindful of your time. You don't need to go overboard, consider what might be necessary to achieve MVP so that this API is easy to understand and consume by a Frontend Developer.
+
+## Things we'll be looking for:
+
+- A strong understanding of Rails
+- Ability to create restful routes
+- Demonstration of well-organized code, following OOP
+- Test Driven Development
+- Clear documentation
+
+Try to limit your total time to approximately 6 hours on this project. Prioritize what is important to completing MVP and demonstrating your capabilities as a developer. If you are looking for a stretch option, you can consume this API for tea information: [Tea API](https://tea-api-vic-lo.herokuapp.com/)
```

Comment:
I like the stretch option included here!  I probably messed up something, but when I did a POST for add a comment, the result was undefined.  Are there some docs for this?

Reply:
Oh interesting, the docs for the API are on the Github page, but to navigate to it you'd need to click the `Victoria Lo` Link at the bottom of the page. I debated on including the link to Github, but since it was a stretch goal I wanted it to be more on the students to look into it rather than provide all the necessary information.